modules/virt-add-disk-to-vm.adoc

```diff
@@ -33,9 +35,9 @@ Use this procedure to add a virtual disk to a {object}.
 
 . Select a {object} to open the *{object-vm-overview}* screen.
 
-. Click the *Disks* tab and then click *Add disk*.
+. On the *{tab}* tab, click *Add disk*.
```

Comment:
The tab value is "Scheduling" from line #22, which is not right as it should be "Disks" tab here, not sure if change line #22 will impact other place, so add a comment here.